finance-tracker/app/views/stocks/_list.html.erb

```diff
@@ -12,7 +12,7 @@
       <tbody>
         <% @tracked_stocks.each do |stock| %>
           <tr>
-            <th scope="row"><%= stock.ticker %></th>
+            <td scope="row"><b><%= stock.ticker %></b></td>
```

Comment:
What is `scope="row"` means?

Reply:
specifies if the header is for a row or column. I will remove this.